### PR TITLE
Inject app_id & app_code from secure environment variables

### PR DESCRIPTION
**Pull request by abdullahselek**
- HERE_SDK_APP_ID and HERE_SDK_APP_CODE env. variables created which get values
from xcodebuild command
- Travis config file updated with additional parameters
- Travis has some current issues so iPhone 6 deleted
- xcpretty added to simplify xcodebuild logs and hide build settings

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+source "https://rubygems.org"
+
+gem "xcpretty"
+gem "cocoapods"
```
Comment (sschuberth) [inline Gemfile:4]: Shouldn't we then have a `Gemfile.lock`, too?

File: .travis.yml [hunk 2]
```diff
@@ -30,7 +32,7 @@ script:
 
 # Build App in Debug and Run Tests if specified
   - if [ $RUN_TESTS == "YES" ]; then
-      travis_retry xcodebuild -workspace "$WORKSPACE" -scheme "$SCHEME" -sdk "$SDK" -destination "$DESTINATION" -configuration Debug ONLY_ACTIVE_ARCH=NO ENABLE_TESTABILITY=YES test;
+      travis_retry xcodebuild -workspace "$WORKSPACE" -scheme "$SCHEME" -sdk "$SDK" -destination "$DESTINATION" -configuration Debug ONLY_ACTIVE_ARCH=NO ENABLE_TESTABILITY=YES APP_ID="$APP_ID" APP_CODE="$APP_CODE" test | xcpretty;
```
Comment (sschuberth) [inline .travis.yml]: BTW, if the trailing `;` is not required here (and in line 37), should we just take the chance to get rid of it?

File: Gemfile (hunk 1 above)
Comment (abdullahselek) [reply Gemfile:4]: Thanks for reminding, done

File: .travis.yml (hunk 2 above)
Comment (abdullahselek) [reply .travis.yml]: I was not sure and I tried we need these `;`

**Review by SirBif — APPROVED** [review]
Looks good

**Pull request merged**